code-build.js

```diff
@@ -205,6 +205,13 @@ function githubInputs() {
       : process.env[`GITHUB_SHA`]);
 
   assert(sourceVersion, "No source version could be evaluated.");
+
+  const sourceTypeOverride = 
+    core.getInput("source-type-override", { required: false, }) || undefined;
+
```

Comment:
These two new attributes need to be added to the returned struct [here](https://github.com/aws-actions/aws-codebuild-run-build/blob/main/code-build.js#L258-L275)